systems/citybuilding.go

```diff
@@ -46,29 +47,44 @@ func (cb *CityBuildingSystem) New(w *ecs.World) {
 // Update is ran every frame, with `dt` being the time
 // in seconds since the last frame
 func (cb *CityBuildingSystem) Update(dt float32) {
-	if engo.Keys.Get(engo.F1).JustPressed() {
-		fmt.Println("The gamer pressed F1")
-
-		city := City{BasicEntity: ecs.NewBasic()}
+	buildKey := engo.Button{[]engo.Key{engo.Q}, "build"}
```

Comment:
This isn't really the right way to use the input manager.

In the new function:
```
engo.Input.RegisterButton("build", engo.Q)
```

and then to retrieve the button:

```
buildKey := engo.Input.Button("build")
```

It probably also makes sense to make `"build"` a constant somewhere